book/checkout.go

```diff
@@ -35,7 +35,9 @@ import (
 func (b *Book) Checkout(step string) error {
 	// FIXME: keep vendor/ node_modules/ around before git clean, but them back as they will be updated the right way, less Internet traffic
 	// FIXME: if the checkout is to a later step, no need to remove the DB, we can just migrate it
-	os.Chdir(b.Dir)
+	if err := os.Chdir(b.Dir); err != nil {
+		return errors.WithStack(err)
+	}
```

Comment:
```suggestion
	if err := errors.WithStack(os.Chdir(b.Dir)); err != nil {
		return err
	}
```

I'm not saying we have to do it, but we can do it when there's only a single return value, this also better preserve stack traces IIRC